### PR TITLE
Saving managed dimension

### PR DESCRIPTION
**Pull request by tpluscode**

### CODE REVIEW

File: apis/core/test/domain/queries/dimension-mappings.test.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import $rdf from 'rdf-ext'
+import { expect } from 'chai'
+import { DESCRIBE } from '@tpluscode/sparql-builder'
+import { parsingClient, client, insertTestData } from '@cube-creator/testing/lib'
+import clownface from 'clownface'
+import TermMap from '@rdfjs/term-map'
+import { replaceValueWithDefinedTerms } from '../../../lib/domain/queries/dimension-mappings'
+
+describe('lib/domain/queries/dimension-mappings @SPARQL', function () {
+  this.timeout(20000)
```
Comment (lucafurrer) [inline apis/core/test/domain/queries/dimension-mappings.test.ts:13]: I like this :smile:

**Review by lucafurrer — APPROVED** [review]
Looks fine to me, but I was not able to find the form in the UI

**Comment by lucafurrer** [comment]
I did some clicking around on this PR and I run into this
image
not sure if it is linked but something to keep in mind.

**Comment by tpluscode** [comment]
Thank you @lucafurrer. This has already been fixed on master

**Comment by tpluscode** [comment]
> I was not able to find the form in the UI

Yes, no form yet

**Pull request merged**